package.json

```diff
@@ -9,17 +9,17 @@
     "express": "^4.17.1",
     "glob": "^7.1.6",
     "merge": "^2.1.0",
-    "swagger-ui-express": "^4.1.5"
+    "swagger-ui-express": "^4.1.6"
```

Comment:
Hi @meabed, thanks for your PR 👏 However, we could not merge it because of this new version. This is a breaking change of the examples. We are dealing with this issue in this [PR](https://github.com/BRIKEV/express-jsdoc-swagger/pull/126) the new version breaks some of the example responses in the Swagger UI.


After we solve this we will merge this. I hope by the end of this month we could do it 🚀 🙌

Reply:
Great, thank you!

Reply:
Hi @meabed the PR that I mentioned was merged 👍 So if you could fix those package.lock errors we could merge this 👍 

Thanks!!!